src/components/info-reveal/index.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Component,
+  h,
+  Prop,
+  State,
+  Watch,
+  Element,
+  Host,
+} from '@stencil/core';
+import { InfoRevealPosition } from "../../beans";
+
+@Component({
+  tag: 'z-info-reveal',
+  styleUrl: 'styles.css',
+  shadow: true
+})
+
+/**
+ * Info reveal component.
+ *
+ * @slot - info to display in the info box. If more than one element has been slotted,
+ * by clicking on the panel it switches to the next info element.
+ * @cssprop --z-info-reveal-theme--surface - background color of the info reveal panel.
+ * @cssprop --z-info-reveal-theme--text - foreground color of the info reveal panel.
+ * @cssprop --z-info-reveal-shadow - shadow of the info reveal panel.
+ * @cssprop --z-info-reveal-max-width - max width of the info reveal panel.
+ */
+export class ZInfoReveal {
+  @Element() el: HTMLElement;
+
+  /** Name of the icon for the open button */
+  @Prop() icon?= 'informationsource';
+
+  /** Position of the open/close button on the panel. */
```

Comment:
Suggerirei di specificare i possibili valori, in modo da avere un'idea chiara dei valori da usare al momento dell'implementazione

es. 'Must be one of "top_left", "top_right", "bottom_left" or "bottom_right". Default is "bottom_right" '